src/ctap/storage.rs

```diff
@@ -31,9 +31,9 @@ use cbor::cbor_array_vec;
 use core::convert::TryInto;
 use crypto::rng256::Rng256;
 
-#[cfg(any(test, feature = "ram_storage"))]
+#[cfg(feature = "std")]
 type Storage = persistent_store::BufferStorage;
-#[cfg(not(any(test, feature = "ram_storage")))]
+#[cfg(not(feature = "std"))]
```

Comment:
Why `std` instead of `test`? Because fuzzing is `std`, but not `test`?

Reply:
Yes exactly. We can't control the `test` condition while we can control features condition like `feature = "std"`. Since tests and fuzzing need `feature = "std"` and production needs `no_std`, this makes a clean distinction between:
 - Production environment: `not(feature = "std")`.
- Testing environment (including fuzzing): `feature = "std"`.

The `test` condition is only needed if something is specific to Rust unit tests, instead of being specific to testing/non-production.

Reply:
Sounds good. I tested the other fuzzing calls, they still work.